README.md

```diff
@@ -1,80 +1,66 @@
-# :electric_plug: `probot-actions-adapter`
+# :electric_plug: `@probot/adapter-github-actions`
 
-> An adapter that takes a [Probot](https://probot.github.io/) app and makes it compatible with [GitHub Actions](https://github.com/features/actions)
+> Adapter to run a [Probot](https://probot.github.io/) application function in [GitHub Actions](https://github.com/features/actions)
 
-<a href="https://github.com/probot/actions-adapter"><img alt="GitHub Actions status" src="https://github.com/probot/actions-adapter/workflows/Build/badge.svg"></a>
-
-## Contents
-
-- [Installation](#installation)
-- [Usage](#usage)
-- [Authentication](#authentication)
-- [Caveats](#caveats)
-
-## Installation
-
-```shell
-npm i -S probot-actions-adapter
-```
+[![Build Status](https://github.com/probot/adapter-github-actions/workflows/Test/badge.svg)](https://github.com/probot/adapter-github-actions/actions)
 
 ## Usage
 
-1. Add an `action.js` to your Probot project, like [the one shown below](#example-actionjs)
-1. Add an `action.yml` to your Probot project, like [the one shown below](#example-actionyml)
-1. _Vendor in_ your `node_modules`, as [recommended by the official Actions documentation](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github)
-1. Optional, but recommended, update your project's README with an example workflow showing how to consume your action
+Create your Probot Application as always
 
-### Example `action.js`
+```js
+// app.js
+module.exports = (app) => {
+  app.on("issues.opened", async (context) => {
+    const params = context.issue({ body: "Hello World!" });
+    await context.octokit.issues.createComment(params);
+  });
+};
+```
 
-```javascript
-// Require the adapter
-const runProbot = require('probot-actions-adapter');
+Then in the entrypoint of your GitHub Action, require `@probot/github-action` instead of `probot`
 
-// Require your Probot app's entrypoint, usually this is just index.js
-const app = require('./index');
+```js
+// index.js
+const { run } = require("@probot/github-action");
+const app = require("./app");
 
-// Adapt the Probot app for Actions
-// This also acts as the main entrypoint for the Action
-runProbot(app);
+run(app).catch((error) => {
+  console.error(error);
+  process.exit(1);
+});
 ```
 
-### Example `action.yml`
+Then use `index.js` as your entrypoint in the `action.yml` file
```

Comment:
There's a bit more nuance here to unpack.

Users can either:

1. Commit their node_modules
2. Compile their JS down to a single executable script (e.g. via `@vercel/ncc` -- this is my preference)

Perhaps we can lean on the docs to unpack this nuance: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github

```suggestion
Follow the official  GitHub documentation for next steps including:

1. [Creating an action metadata file](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-an-action-metadata-file)
1. [Commit, tag, and push your action to GitHub](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github)
```

Reply:
okay like this? https://github.com/probot/adapter-github-actions/pull/19/commits/6760bd2217eefc6060733ace2ac068a6e9adfd9b